frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -487,9 +488,10 @@ export const Chart = ({
 
   const hasDifference = difference && Number.isFinite(difference);
   const disableFilters = !hasData || chartDataMissing;
+  const showFiltersChartPercentageDiff: boolean = chartDataMissing || hasData || !!noDataPlaceholder;
```

Comment:
I'm not sure if it wouldn't be better to add a new boolean prop to the Chart object, smth like "hideChartDetails" that we can set if the lightning account is the only one available, and that can be used instead of `showFiltersChartPercentageDiffs`. The reason is that it looks more maintainable to me, as it is clear what it does and why and it's easier to realize when it's not useful anymore and can be removed. But not a strong opinion, we can also leave it as is. cc @thisconnect wdyt?

Reply:
@Beerosagos I updated to use `hideChartDetails` as a `Chart` boolean prop.